clients/admin-ui/src/features/taxonomy/hooks.tsx

```diff
@@ -164,7 +174,7 @@ export const useDataCategory = (): TaxonomyHookData<DataCategory> => {
     handleEdit,
     handleDelete,
     renderExtraFormFields,
-    transformEntityToInitialValues: transformTaxonomyBaseToInitialValues,
+    transformEntityToInitialValues: handleTransformEntityToInitialValues,
```

Comment:
nit: this'd be less of a potential gotcha if you didn't rename it to `transformEntityToInitialValues` and kept it as `handleTransformEntityToInitialValues`.

Right now that returned function is dangerously close to another locally scoped function, `transformTaxonomyBaseToInitialValues` and I think a mis-type could happen somewhere in the future

Reply:
the consumers of these hooks all expect `transformEntityToInitialValues` (the whole taxonomy page attempts to be very general by putting all the details in the hooks so that it can render data uses, data categories, etc. via the same interface) so I'm hesitant to change it. 

I'm not sure if this is a real pattern, or something that I just picked up somewhere, of the actual `prop` being named some action, and then whatever wrapper the component needs to be named `handle[some action]`. for instance, if a component needs an `onClose`, then for the parent component to call the function locally `handleClose` and then to pass in `onClose={handleClose}`. that's what I was trying to do here, but I understand the confusion. I'll see if there's some way to make it clearer